client/cypress/integration/InfoBase/route_tests.spec.js

```diff
@@ -247,38 +247,37 @@ const route_load_tests_config = [
 
 describe("Route tests", () => {
   route_load_tests_config.map((routes) => {
-    describe(`${routes.name} route`, () => {
+    describe(`${routes.name}`, () => {
       routes.test_on.map((app) => {
-        it(`Tested on ${app}`, () => {
-          cy.on("fail", (e, test) => {
-            if (routes.expect_to_fail) {
-              console.log("Test expected to fail.");
-            } else {
-              throw e;
-            }
-          });
-
+        it(`Tested on index-${app}.html#${routes.route}`, () => {
           cy.visit(
             `http://localhost:8080/build/InfoBase/index-${app}.html#${routes.route}`
           );
-          if (!routes.skipAxe) {
-            if (app == "eng" || app == "fra") {
-              cy.injectAxe();
-              cy.get(".leaf-spinner__inner-circle", { timeout: 10000 }).should(
-                "not.exist"
-              );
-              cy.checkA11y(
-                null,
-                { includedImpacts: ["critical"] },
-                cy.terminalLog,
-                false
-              );
-            }
+
+          if (!routes.skipAxe && (app == "eng" || app == "fra")) {
```

Comment:
Can you remind me why axe is skipped in both basic versions right now? Probably worth capturing in a comment directly on this condition, along with a note on what needs to be done before it can be run on all versions (an issue would be good too, if there isn't already one)